### PR TITLE
Fix incorrect parenthesis of Integral

### PR DESCRIPTION
**Pull request by Ryand1234**
#### References to other Issues or PRs
fixes issue #18758 

#### Brief description of what is fixed or changed
A check for negative integral is added.

#### Release Notes

* printing
   * fixes incorrect parenthesizes in latex print of integral

### CODE REVIEW

File: sympy/printing/latex.py [hunk 1]
```diff
@@ -786,8 +786,16 @@ def _print_Integral(self, expr):
...
+        if any(_coeff_isneg(i) for i in expr.args):
+            return r"%s %s%s" % (tex, self.parenthesize(expr.function,
+                                                    PRECEDENCE["Mul"],
+                                                    is_neg=True,
+                                                    strict=True),
+                             "".join(symbols))
+
         return r"%s %s%s" % (tex, self.parenthesize(expr.function,
                                                     PRECEDENCE["Mul"],
+                                                    is_neg=False,
```
Comment (asmeurer) [inline sympy/printing/latex.py]: This could just be `is_neg=any(_coeff_isneg(i) for i in expr.args)`.

Comment (Ryand1234) [reply sympy/printing/latex.py]: Yes, i committed again with the changes.

**Comment by asmeurer** [comment]
This needs a test.

**Comment by Ryand1234** [comment]
I will add tests once travis-cl completes running.

**Comment by Ryand1234** [comment]
@asmeurer can you please review this PR.

**Comment by Ryand1234** [comment]
Ok

**Pull request merged**